CODEOWNERS

```diff
@@ -7,8 +7,6 @@
 # Add-on maintainers:
 /addons/binding/org.openhab.binding.enocean/ @fruggy83
 /addons/binding/org.openhab.binding.exec/ @kgoderis
-/addons/binding/org.openhab.binding.feed/ @svilenvul
-/addons/binding/org.openhab.binding.feed.test/ @svilenvul
```

Comment:
This one is missing in the CODEOWNERS file

Reply:
fixed